src/pages/home/HomePage.tsx

```
@@ -1,31 +1,20 @@
-      <CtaButton onClick={handleOpen}>시트 열기</CtaButton>
-      <FlipSheet
-        onFlipClick={handleFlip}
-        onChooseClick={handleChoose}
-        isOpen={isSheetOpen}
-        onClose={handleClose}
-      />
+      <h1>Home Page</h1>
+      <TokenRefreshTest />
+      <LogoutButton />
+      <br />
+      <button onClick={() => navigate('/signup')}>Signup</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)